templates/include/glean/event.1.parquetmr.json

```diff
@@ -0,0 +1,23 @@
+{
+  "type": "array",
+  "items": [
+    {
+      "type": "integer",
+      "minimum": 0
+    },
+    {
+      "type": "string"
+    },
+    {
+      "type": "string"
+    },
+    {
+      "type": [ "object", "null" ],
+      "additionalProperties": {
+        "type": [ "string", "null" ]
+      }
```

Comment:
Let's add:

```
"propertyNames": {
  "type": "string",
  "maxLength": 80
}
```

since the keys of the extra properties must be < 80 characters.

Can the extra values actually be `null`?  That doesn't match the existing implementation (but maybe the proposal proposes to change that.  Haven't read it yet :) )

Reply:
No, they can't be - keys for extra properties are now limited to 40 characters.

Reply:
Cool.  Let's make that 40, then, and I assume your PR to android-components will update that there, too.